packages/ts-plugin/src/astro2tsx.ts

```diff
@@ -1,53 +1,49 @@
 import type { FileMapping } from './source-mapper';
-import { Project } from 'ts-morph';
```

Comment:
ts-morph is a super huge dependency, it didn't matter much before, but with all the changes in this PR, it's now twice as big as the entire extension + language-server and we don't use it for much so might as well remove it